### ISSUE TITLE
[ADE Support]: VA Notify : VA Notify Portal: 508 Audit Report Findings - Defect 8 | Critical

### ISSUE THREAD

**Issue by lakper** [department-of-veterans-affairs/va.gov-team#111597]
> [!NOTE]
> **QUICK SUMMARY:** _This ticket is the child ticket for https://github.com/department-of-veterans-affairs/va.gov-team/issues/110877 . The ticket contains information on all of the defects found by the 508 office in relation to WCAG 1.3.1. Some issues were resolved during the first round of feedback, and they have been crossed out below. The team is looking for guidance on how to resolve the remaining issues (4 in total). Links to the staging site where examples of the issues can be found are listed in the ticket._

### Defect 8 | Critical 
_WCAG 1.3.1 Information and Relationships | Information, structure, and relationships conveyed through presentation can be programmatically determined or are available in text._

On the Untitled Notification | VA Notify screen (Accessibility Testing Service 1 -> Add Template -> Email), there is a visual indication that an element is required but this cannot be understood by assistive technology. When tab is used to move the focus to this element, “Edit Button” is read—there is no indication that this is a required element.

New: 05/28/2025
Location: [https://staging.notifications.va.gov/services/42dd098a-7ad6-4e68-999d-9b0b5ce010ba/templates/new/email/content](https://urldefense.com/v3/__https://staging.notifications.va.gov/services/42dd098a-7ad6-4e68-999d-9b0b5ce010ba/templates/new/email/content__;!!K2n62VMvFw!FxwZ1EPdR6LXeJQPKjd4spootk8jry8DAd6PSt0Q8seiOX4UJEEvalQGKqPlBp5NhHbpWsEtj2Wj-P-6TbI-AMOPoto$)

### Ticket Updates
- [ ] _Connect to an `Epic/Parent` or `Intake` (what body of work is this a part of?) If this is the `Epic/Parent`, connect related tickets as sub-issues._
- [ ] _If this is the `Epic/Parent` ticket, add label for `ADE 508 RECHECK`_
- [ ] _If this ticket is a `Child` ticket aka `sub-issue` please add the appropriate labels for `ADE 508 FLAG CRITICAL`;`ADE 508 FLAG HIGH`; `ADE 508 FLAG MEDIUM` or `ADE 508 FLAG LOW`_
- [ ] _Label with `Originator/Team` (product team or stakeholder requesting support; ADE if internal)_
- [ ] _Label date in the `Open Date` field_
- [ ] _Label with `Estimate` (level of effort expected for this ticket)_
- [ ] _Add `Assignee(s)` name(s) to ticket_ (include Lakwi)
- [ ] _Add `Assignee(s) name(s) to each task` they will complete via handle tag (if known)_
- [ ] _Select a `Priority Level`_
- [ ] _Update date in `Last Checked` field_
- [ ] _Label with `Actual` (level of effort it took to complete this ticket)_
- [ ] _Update date in_ `Closed Date`
- [ ] _Make sure the `ADE 508 RECHECK` label is on this ticket upon closing and that a follow-up date has been established and calendared._
